eppo/src/androidTest/AndroidManifest.xml

```diff
@@ -4,5 +4,6 @@
     <application android:usesCleartextTraffic="true" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Pretty sure this is not needed but figure I'd keep it for paranoia, especially in case we start testing old versions of the SDK or something.